app/[locale]/account/connections/page.tsx

```diff
@@ -3,8 +3,11 @@
 import React from 'react';
 import sanitizeHtml from 'sanitize-html';
 import ConnectionSection from '@/components/ConnectionSection';
-import ContentContainer from '@/components/ContentContainer';
-import PageWrapper from '@/components/PageWrapper';
+import PageHeader from '@/components/PageHeader';
+import {
+  LayoutWithPanel,
+  ContentContainer,
+} from '@/components/Container';
```

Comment:
Using new PageHeader, as well as LayoutWithPanel and ContentContainer in page

Reply:
Thanks @andrewebdev. Good point. I think it's because I copied and pasted from my initial profile page that I forgot to use the different LayoutContainer.